src/modes/circle/circle.mode.ts

```diff
@@ -122,6 +123,8 @@ export class TerraDrawCircleMode extends TerraDrawBaseDrawMode<CirclePolygonStyl
 			this.clickCount++;
 			this.setDrawing();
 		} else {
+			if (document.documentElement.classList.contains("is-mobile"))
```

Comment:
This line is too specific to the mapping library you're using - we need a more agnostic way to approach this! Let's have a conversation about how that might work in #106

Reply:
I got this information from the file main.js
https://github.com/JamesLMilner/terra-draw/blob/02937d041669ee1899768ac9fdfd197b15ac8622/docs/assets/main.js#L1999

Reply:
So this is the from the generated docs file, not from the source or distribution file of the library. We don't apply this class anywhere with the actual library code itself.